src/main/java/fi/fmi/avi/converter/tac/conf/TACConverter.java

```diff
@@ -115,6 +116,12 @@
     public static final ConversionSpecification<GenericMeteorologicalBulletin, String> GENERIC_BULLETIN_POJO_TO_TAC = new ConversionSpecification<>(
             GenericMeteorologicalBulletin.class, String.class, null, "WMO GTS bulletin");
 
+    /**
+     *
+     */
+    public static final ConversionSpecification<String, GenericAviationWeatherMessage> TAC_TO_GENERIC_AVIATION_WEATHER_MESSAGE_POJO =
+            new ConversionSpecification<>(String.class, GenericAviationWeatherMessage.class, "TAC string", null);
```

Comment:
I would use `ICAO Annex 3 TAC` as output specifier, just like in other message conversions.